### DISCUSSION TITLE
关于自定义域名页面显示不正常，需要如何正确放置CNAME文件

### DISCUSSION THREAD

**Discussion by s-theo** [vuepress-theme-hope/vuepress-theme-hope#3431] [Q&A] [answered]
我需要如何正确放置CNAME文件？或者哪里需要修改？
`https://<USERNAME>.github.io/<REPO>/` ，将 base 设置为 `"/<REPO>/"`。

这是未添加域名时可以正常访问
![](https://cdn.jsdelivr.net/gh/vanhiupun/pic@1.0/img/202309050452847.png)

这是添加域名后，页面却显示不正常
![](https://cdn.jsdelivr.net/gh/vanhiupun/pic@1.0/img/202309050453105.png)

**Comment by Mister-Hope** [comment, marked as answer]
添加域名之后base应该是/
